start from scratch backend/src/routes/projects.js

```
@@ -1,14 +1,119 @@
+                    // Calculate total instructed spend
+                    instructedSpend: {
+                        $reduce: {
+                            input: '$instructedQuotes',
+                            initialValue: 0,
+                            in: {
+                                $add: [
+                                    '$$value',
+                                    { 
+                                        $cond: {
+                                            if: { $eq: ['$$this.instructionStatus', 'instructed'] },
+                                            then: '$$this.total',
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)